docs/reuse/install-rtu.txt

```diff
@@ -0,0 +1,4 @@
+Refer to `Ubuntu Pro Client - How to enable Real-time Ubuntu`_ to set up a
```

Comment:
i added "Ubuntu Pro Client" to make it a bit more obvious that these would be bringing the user out of the current docs set.